mp/src/game/client/neo/ui/neo_hud_ghost_beacons.h

```diff
@@ -6,20 +6,24 @@
 
 #include "neo_hud_childelement.h"
 #include "hudelement.h"
-#include <vgui_controls/Panel.h>
+#include <vgui_controls/EditablePanel.h>
 
 #include "weapon_ghost.h"
 
-class CNEOHud_GhostBeacons : public CNEOHud_ChildElement, public CHudElement, public vgui::Panel
+class CNEOHud_GhostBeacons : public CNEOHud_ChildElement, public CHudElement, public vgui::EditablePanel
 {
-	DECLARE_CLASS_SIMPLE(CNEOHud_GhostBeacons, Panel);
+	DECLARE_CLASS_SIMPLE(CNEOHud_GhostBeacons, EditablePanel);
 
 public:
 	CNEOHud_GhostBeacons(const char *pElementName, vgui::Panel *parent = NULL);
+	virtual void ApplySchemeSettings(vgui::IScheme* pScheme);
 
 	virtual void Paint();
 	void DrawPlayer(const Vector& playerPos) const;
 
+	Vector ghostBeaconOffset = Vector(0, 0, 0);
+	Vector* m_pGhostBeaconOffset = &ghostBeaconOffset;
```

Comment:
What's with having two variables which one just points to the other when a single `Vector` suffice?